react/package.json

```diff
@@ -11,15 +11,20 @@
     "author": "hey@posthog.com",
     "license": "MIT",
     "scripts": {
-        "clean": "rimraf -r dist",
         "prebuild": "npm run clean",
-        "build": "tsc",
         "start": "tsc-watch",
-        "test": "jest"
+        "clean": "rimraf -r dist",
+        "build": "yarn clean && cross-env NODE_ENV=production rollup -c rollup.config.ts",
+        "test": "cross-env BABEL_ENV=test jest --runInBand",
```

Comment:
Could you please elaborate on the purpose of `cross-env BABEL_ENV=test` here?

Reply:
Using it in other projects to ensure babel is running in the `test`-environment. I have removed it.